lib/hammer_cli_katello/output/fields.rb

```diff
@@ -10,6 +10,9 @@ class ChecksumFilePair < ::Fields::Field
       class Dependency < ::Fields::Field
       end
 
+      class Array < ::Fields::Field
```

Comment:
That makes sense. It will require a PR into hammer-cli though. I'll attach a link once the PR is ready.